hurma/hive_sql/create_schema.hql

```diff
@@ -0,0 +1,83 @@
+create database if not exists crunchbase;
```

Comment:
which schema? naming

Reply:
https://stackoverflow.com/questions/22144685/create-a-schema-in-hive
The uses of SCHEMA and DATABASE are interchangeable – they mean the same thing